### ISSUE TITLE
Send email on error

### ISSUE THREAD

**Issue by tillsteinbach** [tillsteinbach/VWsFriend#133]
As it might happen that you need to take action it would be good to get an email notification on errors.

**Comment by tillsteinbach** [comment]
Feature is now in testing

**Comment by tillsteinbach** [comment]
This is released with 0.14.0:
```
Logging to Email (Errors only):
  --logging-mail-from LOGGINGMAILFROM
                        Mail address to send mails from
  --logging-mail-to LOGGINGMAILTO
                        Mail address to send mails to (can be used multiple
                        times)
  --logging-mail-host LOGGINGMAILHOST
                        Mail server host
  --logging-mail-credentials LOGGINGMAILCREDENTIALS LOGGINGMAILCREDENTIALS
                        Mail server credentials
  --logging-mail-subject LOGGINGMAILSUBJECT
                        Mail subject
  --logging-mail-notls  Mail do not use TLS
  --logging-mail-testmail
                        Try to send Testmail at startup
```
